src/components/InputTime/InputTime.test.tsx

```diff
@@ -33,6 +33,86 @@ describe('InputTime', () => {
     expect(baseInputTime.value).toBe('');
   });
 
+  describe.only('dropdown time picker', () => {
```

Comment:
I believe you forgot to remove the `.only` calls here

Reply:
Maybe add the `eslint` rule to avoid committing these?